### PR TITLE
Cleaning up Stylesheets

### PR DESCRIPTION
**Pull request by xtine**
- Created folders for comment and font icon related stylesheets
- Made tab spacing consistent across all stylesheets
- Cleaned up stylesheets with some scss-lint rules

### CODE REVIEW

**Comment by donjo** [comment]
I sense that this is going to be hard to merge in after the color variable changes or vice versa. Would it make sense to merge the color vars stuff in first then I can run the linters on that and break it out into the file/folder structure you've established here in a separate PR?

**Comment by cmc333333** [comment]
Thank you thank you thank you for breaking this out into changesets. They make this much easier to digest.

**Comment by xtine** [comment]
@donjo: yeah merge your color vars in first. Through looking at every single stylesheet I also came across several places where the color hex codes were used instead of a color variable. That should probably be the step after this one.

**Comment by xtine** [comment]
@donjo: I merged in the updated color variables and fixed conflicts.

File: regulations/static/regulations/css/scss/typography.scss [hunk 1]
```diff
@@ -245,7 +247,7 @@ a:visited {
 .external:focus,
 .mail:focus,
 .go:focus,
-.pdf:focus
+.pdf:focus,
```
Comment (cmc333333) [inline regulations/static/regulations/css/scss/typography.scss]: Nice catch!

File: regulations/static/regulations/css/scss/module/about.scss [hunk 2]
```diff
@@ -46,37 +46,21 @@ hero
...
-
-/*
-layout
--------------------
-*/
-
-
-.about-section {
-  padding: 4em 0;
+  padding: 4em 0 1em 0;
```
Comment (cmc333333) [inline regulations/static/regulations/css/scss/module/about.scss]: There used to be two rules:
```css
padding-bottom: 1em;
// later
padding: 4em 0;
```

Wouldn't the first have been ignored? If so, this new rule isn't the same as what existed previously:
```
padding: 4em 0 1em 0;
```

File: regulations/static/regulations/css/scss/main.scss [hunk 3]
```diff
@@ -304,8 +308,8 @@ Clearfix
 */
 .group:before,
 .group:after {
-  content: " "; /* 1 */
-  display: table; /* 2 */
```
Comment (cmc333333) [inline regulations/static/regulations/css/scss/main.scss]: I wonder what "1" and "2" meant

**Comment by cmc333333** [comment]
I ran over the changesets -- only bit that I caught was a potential padding issue. Otherwise, I don't see anything missing.

I'm not a fan of the underscore file names, but if we must, we must.

**Comment by donjo** [comment]
looks good! great job 👍

**Pull request merged**